recipe/meta.yaml

```diff
@@ -15,30 +15,37 @@ source:
   {{ hash_type }}: {{ hash_val }}
 
 build:
-  noarch: python
-  number: 1
+  number: 0
   script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<37]
 
 requirements:
-  build:
+  host:
     - python
     - pip
+    - setuptools
+    - wheel
 
   run:
     - python
     - flask
-    - bcrypt
+    - bcrypt >=3.1.1
 
 test:
   imports:
     - flask_bcrypt
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
-  home: https://github.com/maxcountryman/flask-bcrypt
+  home: https://flask-bcrypt.readthedocs.io/en/1.0.1/
```

Comment:
The correct home URL is ``https://github.com/maxcountryman/flask-bcrypt` https://github.com/maxcountryman/flask-bcrypt/blob/d9064521998846a59a98f311e3bd135e1dfe082c/setup.py#L17

Reply:
Ahhh okay, thanks for pointing that out! I'll be sure to check there for the home urls.